### DISCUSSION TITLE
Dockert Socket

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#4435] [Questions] [answered]
https://github.com/nextcloud/all-in-one/blob/10698d7df08fe937a03df263a342f09abdc31824/Containers/mastercontainer/start.sh#L50
This line says it checks for the group id, but the next line only checks for the group name which doesn't seem to make sense, or am I wrong?

**Comment by szaimen** [comment, marked as answer]
The comment is wrong. It should be group name not group id

**Reply by szaimen** [reply]
https://github.com/nextcloud/all-in-one/commit/55d697166811382ab21ecc5fedb7d0ad446f99bd
